packages/pentaho/src/Canvas/SidePanel/SidePanel.styles.tsx

```diff
@@ -50,6 +50,7 @@ export const { staticClasses, useClasses } = createClasses(
       },
       "&:hover": {
         cursor: "pointer",
+        backgroundColor: `color-mix(in srgb, ${theme.palette.blue[600]} 10%, transparent)`,
```

Comment:
We're slowly re-implementing all of `HvIconButton`/`HvButton` unnecessarily. We should reuse `HvIconButton` for the icon

Reply:
Yeah, I removed the redundant styles.